src/converter/one/jfr/Dictionary.java

```diff
@@ -16,13 +18,17 @@ public class Dictionary<T> {
     private int size;
 
     public Dictionary() {
-        this.keys = new long[INITIAL_CAPACITY];
-        this.values = new Object[INITIAL_CAPACITY];
+        this(INITIAL_CAPACITY);
+    }
+
+    public Dictionary(int initialCapacity) {
+        this.keys = new long[initialCapacity];
+        this.values = new Object[initialCapacity];
     }
 
     public void clear() {
-        keys = new long[INITIAL_CAPACITY];
-        values = new Object[INITIAL_CAPACITY];
+        Arrays.fill(keys, 0);
+        Arrays.fill(values, null);
```

Comment:
I think because of profiling) But it is pretty hard to say now. I'll change it in DictionaryInt as well, if you are ok, but I also can try revert this one, I'm not sure if it is critical.